ca/config.go

```diff
@@ -274,6 +274,9 @@ func LoadSecurityConfig(ctx context.Context, rootCA RootCA, krw *KeyReadWriter)
 
 	// Check to see if this certificate was signed by our CA, and isn't expired
 	if _, err := X509Cert.Verify(opts); err != nil {
+		fmtString := "Jan 02 15:04:05 2006 MST"
+		log.G(ctx).WithError(err).Debugf("invalid certificate issued by %s (not before %s, not after %s)",
```

Comment:
Have removed the log message, since it gets logged elsewhere, and am just returning the timestamps in the error.